test/unit/models.spec.js

```diff
@@ -0,0 +1,181 @@
+import { afterEach, beforeEach, describe, expect, it, jest } from '@jest/globals';
+import { getModelInheritanceChain, getModelConfig, getModelContainer } from '../../js/models';
+
+jest.mock('core/js/adapt', () => ({
+  __esModule: true
+}));
+jest.mock('core/js/components', () => ({
+  __esModule: true
+}));
+jest.mock('core/js/data', () => ({
+  __esModule: true,
+  default: { isReady: true }
+}));
+jest.mock('core/js/logging', () => ({
+  __esModule: true
+}));
+jest.mock('core/js/models/contentObjectModel', () => ({
+  __esModule: true
+}));
+jest.mock('core/js/models/courseModel', () => ({
+  __esModule: true
+}));
+
+class MockModel extends Backbone.Model {
+  getParent() { return this.get('_parent'); }
+}
+
+describe('getModelInheritanceChain', () => {
+  describe('when given an article', () => {
+    it('should return only the article in an array', () => {
+      const article = new MockModel({ _type: 'article' });
+
+      expect(getModelInheritanceChain(article)).toStrictEqual([article]);
+    });
+  });
```

Comment:
This isn't unique to Trickle. Should the test be written into Core instead?

Reply:
The function only exists in trickle.

Reply:
This function only exists in trickle's models.js though?

Reply:
Ah, I misread the import. Apologies